Classes/book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+  end
```

Comment:
- [ ] `*the book` itself should be sent as a parameter to create the new rental achieving the 'has-many' association. a new rental should be created with `date`, `person` and `book` itselff

Reply:
Do you mean writing  
rentals.book =self